### ISSUE TITLE
[Bug] Upload notes endpoint returns 500

### ISSUE THREAD

**Issue by aniri** [code4romania/monitorizare-vot#146]
A call to the following endpoint:

`https://mv-mobile-test.azurewebsites.net/api/v2/note/upload`

returns 500.

```
curl -X POST "https://mv-mobile-test.azurewebsites.net/api/v2/note/upload" -H  "accept: application/json" -H  "Authorization: Bearer XXXXXX" -H  "Content-Type: multipart/form-data" -F "CountyCode=AB" -F "PollingStattionNumber=2" -F "Text=test note"
```

**Comment by idormenco** [comment]
I will investigate and try to fix

**Comment by RaduCStefanescu** [comment]
In progress @idormenco

**Comment by RaduCStefanescu** [comment]
Fixed in #148
